src/utils.py

```diff
@@ -88,8 +87,54 @@ def load_or_create_measurements_df(model, model_name, train_timesteps, file_path
         measurements = model.get_measurements()
         df = pd.DataFrame(measurements)
         df['state'] = 'train' # makes it easier to later understand which part was training
-        df.loc[df['timesteps'] > train_timesteps, 'state'] = 'run'
+        df.loc[measurements_df['timesteps'] > train_timesteps, 'state'] = 'run'
         df['model'] = model_name
-        df['lambda'] = globals.ALPHA
     
-    return df
\ No newline at end of file
+    return df
+
+def user_topic_mapping(user_profiles, item_attributes, item_topics):
```

Comment:
This would change when using Prof.Zemel's idea of creating clusters for both users and items right?